integration-tests/main/src/test/java/io/quarkus/it/main/BeanManagerInjectionInTestCase.java

```diff
@@ -18,6 +18,7 @@ public class BeanManagerInjectionInTestCase {
     @Test
     public void testInjection() {
         Assertions.assertNotNull(beanManager);
-        Assertions.assertFalse(beanManager.getBeans(ExternalService.class).isEmpty());
+        Assertions.assertFalse(beanManager.getBeans(ExternalService.class).isEmpty(),
+                () -> "Beans is " + beanManager.getBeans(ExternalService.class));
```

Comment:
I know it's probably not necessary in this case but if I were you I would create a variable. If the return value is not stable, you might have issues.

(Now, I'm pretty sure it's not the case here but it doesn't hurt to be a bit cautious)

Reply:
Makes sense - will update :)